package.json

```diff
@@ -22,44 +22,26 @@
     "react",
     "react-component"
   ],
+  "peerDependencies": {
+    "react": "*",
+    "react-native": "*"
+  },
   "devDependencies": {
-    "babel": "^6.3.13",
-    "babel-eslint": "^4.1.4",
-    "babel-plugin-external-helpers-2": "^6.1.4",
-    "babel-plugin-syntax-async-functions": "^6.0.14",
-    "babel-plugin-syntax-class-properties": "^6.0.14",
-    "babel-plugin-syntax-flow": "^6.0.14",
-    "babel-plugin-syntax-jsx": "^6.0.14",
-    "babel-plugin-syntax-trailing-function-commas": "^6.0.14",
-    "babel-plugin-transform-class-properties": "^6.0.14",
-    "babel-plugin-transform-es2015-arrow-functions": "^6.0.14",
-    "babel-plugin-transform-es2015-block-scoping": "^6.0.18",
-    "babel-plugin-transform-es2015-classes": "^6.1.2",
-    "babel-plugin-transform-es2015-computed-properties": "^6.0.14",
-    "babel-plugin-transform-es2015-constants": "^6.0.15",
-    "babel-plugin-transform-es2015-destructuring": "^6.0.18",
-    "babel-plugin-transform-es2015-for-of": "^6.0.14",
-    "babel-plugin-transform-es2015-modules-commonjs": "6.26.0",
-    "babel-plugin-transform-es2015-parameters": "^6.0.18",
-    "babel-plugin-transform-es2015-shorthand-properties": "^6.0.14",
-    "babel-plugin-transform-es2015-spread": "^6.0.14",
-    "babel-plugin-transform-es2015-template-literals": "^6.0.14",
-    "babel-plugin-transform-flow-strip-types": "^6.0.14",
-    "babel-plugin-transform-object-assign": "^6.0.14",
-    "babel-plugin-transform-object-rest-spread": "^6.0.14",
-    "babel-plugin-transform-react-display-name": "^6.0.14",
-    "babel-plugin-transform-react-jsx": "^6.0.18",
-    "babel-plugin-transform-regenerator": "^6.0.18",
-    "babel-register": "^6.3.13",
-    "chai": "^3.4.0",
-    "eslint": "^1.8.0",
-    "eslint-plugin-react": "^3.7.1",
-    "mocha": "^2.3.3"
+    "@babel/core": "7.1.5",
+    "babel-core": "^7.0.0-bridge.0",
+    "babel-jest": "23.6.0",
+    "jest": "23.6.0",
+    "metro-react-native-babel-preset": "0.49.0",
+    "react-native": "0.57.4",
```

Comment:
I'm confused -- these are indeed in the `devDependencies`? -- These are the things that are used for development & testing of this repo. -- To my knowledge, this library is compatible with any version of react/react-native, so the `peerDependencies` have those wild carded out.

These *shouldn't* be in regular `dependencies` because those are "master" libraries and we wouldn't want the user to end up installing multiple copies of `react-native`, for example